Android/app/src/main/res/layout/activity_home.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
+<androidx.coordinatorlayout.widget.CoordinatorLayout
```

Comment:
Whats the use of a coorindator layout here, why is this better than a constraint layout?

Reply:
Had to use it for the bottom sheet, but now I dont think it matters